ReactiveCocoa/Objective-C/RACPassthroughSubscriber.m

```diff
@@ -75,7 +84,9 @@ - (void)sendNext:(id)value {
 		RACSIGNAL_NEXT(cleanedSignalDescription(self.signal), cleanedDTraceString(self.innerSubscriber.description), cleanedDTraceString([value description]));
 	}
```

Comment:
The dtrace macros should probably be moved inside the lock?

And I think this means that the check of `self.disposable.disposed` can be removed. It should probably be replaced with an early exited that checks `if (self.innerSubscriber == nil) return;`.